crates/walrus-service/src/event/event_processor/processor.rs

```diff
@@ -147,13 +176,23 @@ impl EventProcessor {
 
         let clients = client_manager.into_client_set();
 
+        let (coordination_state, _coordination_rx) = CatchupCoordinationState::new();
+        let coordination_state = Arc::new(coordination_state);
+
         let catchup_manager = EventBlobCatchupManager::new(
             event_processor.stores.clone(),
             clients,
             system_config,
             runtime_config.db_path.join("recovery"),
             metrics_registry,
+            catchup::CatchupRuntimeConfig {
+                coordination_state: coordination_state.clone(),
+                coordination_timeout: Duration::from_secs(STARTUP_CATCHUP_TIMEOUT_SECS),
+                processing_timeout: Duration::from_secs(STARTUP_CATCHUP_TIMEOUT_SECS),
+            },
         );
+        coordination_state.notify_tailing_stopped();
+
         if let Err(e) = catchup_manager
             .catchup(config.event_stream_catchup_min_checkpoint_lag)
```

Comment:
Suggestion: can we create EventBlobCatchupManager only if we need to perform catchup using event blob?

Reply:
This should be not needed now